### ISSUE TITLE
Waiting for victim to forgive or keep report

### ISSUE THREAD

**Issue by Quinlan102** [Tommy228/TTTDamagelogs#334]
In my opinion I think it's too hard to tell if the report is still active and waiting for staff to action it or if the victim still needs to forgive or keep report.
it would be alot better if it was more clear visually

The reason I'm bringing this up because I've had a lot of staff do reports just to have the victim forgive the report player.

**Comment by iagrib** [comment]
I agree. I think displaying something different from red cross icon before the victim has decided to keep or cancel the report is the way to go.

**Comment by Phoenixmeister** [comment]
I wanted to mention it for weeks now, and i hope it is going to be done like by the old damagelogs. That is the only problem with the new one and that the deathscene is sometimes not shown properly

**Comment by ThisisKadse** [comment]
The Supportes on my Server have some problems to see which report is allready canceled and which one is open and undone. Please change this thing by the next update, that whould be very nice.

**Comment by MeowZedong94** [comment]
Perhaps adding a ? to the report when the attacker responds would be a good idea?
